rts/Lua/LuaHandle.cpp

```diff
@@ -2392,6 +2392,12 @@ DRAW_CALLIN(DrawPreDecals)
  */
 DRAW_CALLIN(DrawWorldPreParticles)
 
+/*** This callin has z and color buffer of shadowmap bound via FBO as well as the FFP state to do "semi-transparent" shadows pass (traditionally only used to draw particles).
```

Comment:
Just a minor remark. Use the text above `@function` for the one-liner descriptor. Below for extended description, e.g.:

```cpp
/*** Invoked when xyz
 *
 * @function Whatever
 * 
 * Does blablabl et blablabla, when lorem ipsum. Beware of harberblabber.
 */
```

Reply:
Ah ok, I was copying the syntax from somewhere else. Thanks for the hint.

Reply:
Yeah, currently the syntax is a bit over the place as it's an extended effort. It's not a problem, but once I work on the generator the one-liners will be more important (they are already used in the horrible default layout).